src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -909,6 +909,7 @@ public sealed partial class SqlConnectionStringBuilder : System.Data.Common.DbCo
         /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/AsynchronousProcessing/*'/>
         [System.ComponentModel.DisplayNameAttribute("Asynchronous Processing")]
         [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        [System.ObsoleteAttribute("AsynchronousProcessing has been deprecated.  SqlConnection will ignore the 'Asynchronous Processing' keyword and always allow asynchronous processing the connection.")]
```

Comment:
Updating to match below.
```suggestion
        [System.ObsoleteAttribute("AsynchronousProcessing has been deprecated.  SqlConnection will ignore the 'Asynchronous Processing' keyword and always allow asynchronous processing.")]
```